data/sources/unassigned.json

```diff
@@ -1,6 +1,5 @@
 {
   "164083100": "Source: Display of Supremacy, Weekly Challenge",
-  "2939318890": "Source: Wayfinder's Compass",
   "3094114967": "Source: Season of the Lost ritual playlists",
```

Comment:
is this just shattered realm and astral alignment?

Reply:
3094114967? i dont know this is weird. the terminology has gotten strange over there..... it's definitely the set of seasonal gear but not sure exactly what they are trying to imply with the source string